setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+VERSION = 1
+
+setup(
```

Comment:
List dependencies as laxly as possible, i.e., no `==` and instead `>=` if version specifiers are truly needed.